### ISSUE TITLE
Org Mode Reader duplicates footnotes

### ISSUE THREAD

**Issue by edom** [jgm/pandoc#5196]
# How to reproduce

Pandoc version

```
pandoc 2.4
Compiled with pandoc-types 1.17.5.1, texmath 0.11.1.2, skylighting 0.7.4
```

Command line

```
pandoc a.org -o a.html
```

Input file `a.org`

```
First reference[fn:A].

Second reference[fn:A].

Sole reference[fn:B:Footnote B.].

[fn:A]
Footnote A.
```

Actual output:

```
<p>First reference<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>.</p>
<p>Second reference<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>.</p>
<p>Sole reference<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a>.</p>
<section class="footnotes">
<hr />
<ol>
<li id="fn1"><p>Footnote A.<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>Footnote A.<a href="#fnref2" class="footnote-back">↩</a></p></li>
<li id="fn3"><p>Footnote B.<a href="#fnref3" class="footnote-back">↩</a></p></li>
</section>
```

Expected output (Note that "Footnote A" occurs only once instead of twice. Also note that the backreferences are letters if there are many backreferences, but the backreference is an arrow if there is only one backreference.):

```
<p>First reference<a href="#fn1" class="footnote-ref" id="fnref1_0"><sup>1</sup></a>.</p>
<p>Second reference<a href="#fn1" class="footnote-ref" id="fnref1_1"><sup>1</sup></a>.</p>
<p>Sole reference<a href="#fn2" class="footnote-ref" id="fnref2_0"><sup>2</sup></a>.</p>
<section class="footnotes">
<hr />
<ol>
<li id="fn1"><p>Footnote A.<a href="#fnref1_0" class="footnote-back">a</a> <a href="#fnref1_1" class="footnote-back">b</a></p></li>
<li id="fn2"><p>Footnote B.<a href="#fnref2_0" class="footnote-back">↩</a></p></li>
</section>
```

For comparison, [Wikipedia](https://en.wikipedia.org/wiki/Haskell_(programming_language)#References) has output similar to the expected output. Please feel free to move the backreferences to the front (following Wikipedia) or keep the backreferences at the back (the current state of Pandoc).

Org mode version 9.1.14 with body-only export to plain text (C-c C-e C-b t U) also has output similar to the expected output (although Pandoc does backreferences better than Org Mode does).
But please don't follow the Org Mode output completely. (We don't want a "Footnotes: " heading, do we?)
The following is Org Mode output:

```
First reference[1].

Second reference[1].

Sole reference[2].



Footnotes
─────────

[1] Footnote A.

[2] Footnote B.
```

**Comment by tarleb** [comment]
Thanks for the report. This is a problem common to all formats. The underlying reasons are described in #1603 and #224. Closing in favor of  issue #1603.

**Comment by tarleb** [comment]
If you are using footnotes for references, then maybe take a look at John Kitchen's org-ref package. It can be combined with pandoc.
